x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/scalar/nulls/Coalesce.java

```diff
@@ -213,7 +213,7 @@ public ExpressionEvaluator.Factory toEvaluator(ToEvaluator toEvaluator) {
             case EXPONENTIAL_HISTOGRAM -> CoalesceExponentialHistogramEvaluator.toEvaluator(toEvaluator, children());
             case NULL -> EvalOperator.CONSTANT_NULL_FACTORY;
             case UNSUPPORTED, SHORT, BYTE, DATE_PERIOD, OBJECT, DOC_DATA_TYPE, SOURCE, TIME_DURATION, FLOAT, HALF_FLOAT, TSID_DATA_TYPE,
-                SCALED_FLOAT, AGGREGATE_METRIC_DOUBLE, TDIGEST, DENSE_VECTOR -> throw new UnsupportedOperationException(
+                SCALED_FLOAT, AGGREGATE_METRIC_DOUBLE, TDIGEST, HISTOGRAM, DENSE_VECTOR -> throw new UnsupportedOperationException(
```

Comment:
We should get to stuff like this before long.

I wonder if there's a good way of "querying" the code base for the list of outstanding "stuff" for new data types. Like this.

Reply:
Anything like that would need a way to differentiate between "real" types (like Histogram) and types that should not exist by this point in the plan, like Half Float.

That said, I intend to implement this for Histogram (and probably T-Digest, unless @JonasKunz gets to it first) soon.  Just, not yet.